base/channels.jl

```diff
@@ -95,6 +95,22 @@ take!(c) = "Hello"
 julia> istaskdone(taskref[])
 true
 ```
+
```

Comment:
I think there is a `!!! note` for stating the Julia version since when a change is valid (this will be 1.2)

Reply:
`!!! compat "Julia 1.2"`

Reply:
so just add this at the bottom of the comment, right here?:
```suggestion
!!! compat "Julia 1.2"
```

Reply:
Yea, see the note in CONTRIBUTING.md and other examples in the code base.

Reply:
Thanks @fredrikekre. Sorry this took me so long to do.

I've added a compat note for these new constructors and the new default constructor that I just added, `Channel()`.